src/CbcCInterface.jl

```diff
@@ -214,21 +214,25 @@ end
 @getproperty Float64 getObjSense
 
 for s in (:getRowLower, :getRowUpper, :getRowActivity)
-    @eval function ($s)(prob::CbcModel)
+    unsafe_s = Symbol("unsafe", '_', s)
+    @eval function ($unsafe_s)(prob::CbcModel)
```

Comment:
Please add comments explaining why this is unsafe (i.e., It returns an array whose memory is owned by Cbc. The underlying memory could be invalidated whenever Cbc feels like it.)